src/AppBundle/Controller/SwindlerController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use Sonata\AdminBundle\Controller\CRUDController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+class SwindlerController extends CRUDController
+{
+    /**
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function lockAction()
+    {
+        $performanceEventId = $this->get('request')->query->get('id');
```

Comment:
yes, $performanceEventId - copy paste. 
Maybe instead param convertor use 
$this->admin
            ->getSubject()
           ->setBanned(true);
        $em = $this->getDoctrine()->getManager();
        $em->flush($swindler);